spec/sidekiq-scheduler/schedule_spec.rb

```diff
@@ -29,24 +31,34 @@ def job_from_redis_without_decoding(job_id)
   let(:job_class_id) { cron_hash['class'] }
 
   describe '.schedule=' do
-    it 'sets the schedule on redis' do
-      Sidekiq::Scheduler.dynamic = true
+    before { Sidekiq::Scheduler.dynamic = true }
 
+    it 'sets the schedule on redis' do
       Sidekiq.schedule = {job_id => cron_hash}
 
       expect(cron_hash).to eq(job_from_redis(job_id))
     end
 
     it 'uses job name as \'class\' argument if it\'s missing' do
-      Sidekiq::Scheduler.dynamic = true
-
       Sidekiq.schedule = {job_class_id => cron_hash.select(&only_cron_and_args)}
 
       job = Sidekiq.schedule[job_class_id]
 
       expect(cron_hash).to eq(job_from_redis(job_class_id))
       expect(job['class']).to eq(job_class_id)
     end
+
+    context 'when job key is a symbol' do
+      let(:job_id) { :super_job }
+
+      context 'when schedule is set twice' do
+        it 'sets the schedule on redis' do
```

Comment:
This test does not fail when running the specs with seed = 9838. Please flush Redis on a `before` hook (see `spec/sidekiq/scheduler_spec.rb`)